src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -15,6 +16,8 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
 {
     public class NativeIntTests : CSharpTestBase
     {
+        private static readonly SymbolDisplayFormat FormatWithSpecialTypes = SymbolDisplayFormat.TestFormat.WithMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
+
         [Fact]
```

Comment:
>FromMetadata [](start = 20, length = 12)

It doesn't look like the test tests metadata.

Reply:
Renamed the tests and added comments, as we discussed.

---
In reply to: [374858980](https://github.com/dotnet/roslyn/pull/40994#discussion_r374858980) [](ancestors = 374858980)